nginx/kobo-docker-scripts/nginx.conf

```diff
@@ -50,8 +50,25 @@ http {
     # Gzip Settings
     #####################
 
-    gzip on;
-    gzip_disable "msie6";
+        # gzip configs from here
+        # http://stackoverflow.com/a/12644530/3088435
+        gzip on;
+        gzip_disable "msie6";
+        gzip_comp_level 6;
+        gzip_min_length 1100;
+        gzip_buffers 16 8k;
+        gzip_proxied any;
+        gzip_types
+            text/plain
+            text/css
+            text/js
+            text/xml
+            text/javascript
+            application/javascript
+            application/x-javascript
+            application/json
+            application/xml
+            application/xml+rss;
```

Comment:
As I suggested, we should add svg too  (i.e. `image/svg+xml`)

Reply:
I added image/svg+xml to the `gzip_types` list